launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -71,6 +72,10 @@ class ForkScaffolder(private val launcher: Launcher) {
                             label = "Author HTTP URL"
```

Comment:
this should also cover publish instance

toggle if author or publish instance url is ending with that ...